database/models.py

```diff
@@ -0,0 +1,110 @@
+from sqlalchemy import (
+    create_engine,
+    Column,
+    Integer,
+    String,
+    Float,
+    JSON,
+    BigInteger,
+    Boolean,
+)
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+import os
+from logger.logger import logger
+
+# Get database URL from environment variable
+DATABASE_URL = os.getenv("DATABASE_URL")
```

Comment:
might be better to just assign all the environment variables to vars up here (and in the case of "TESTING" covert it to a boolean type). also could be good to through an error here if required envars aren't defined